main.go

```
@@ -0,0 +1,155 @@
+							response.Text = fmt.Sprintf("%s \n %s", a.Title, a.URL)
+							_, err := bot.Send(response)
+							if err != nil {
+								logger.Error(err)
+								continue
+							}
+						}
+						continue
+					default:
+						response.Text = "sorry, i don't know the command"
+					}
+					bot.Send(response)
```

golangci-lint (1.62.2)

Error return value of `bot.Send` is not checked

(errcheck)